apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -144,14 +167,23 @@ module.exports = class TestTab extends ViewPlugin {
       this.testsOutput.appendChild(this.outputHeader)
     } else if (result.type === 'testPass') {
       this.testsOutput.appendChild(yo`
-        <div id="${this.runningTestFileName}" data-id="testTabSolidityUnitTestsOutputheader" class="${css.testPass} ${css.testLog} bg-light mb-2 text-success border-0">
+        <div
+          id="${this.runningTestFileName}"
+          data-id="testTabSolidityUnitTestsOutputheader"
+          class="${css.testPass} ${css.testLog} bg-light mb-2 text-success border-0"
+          onclick=${() => this.discardHighlight()}
```

Comment:
why do you `discardHighlight` when one click on a successful test?

Reply:
feels better like this. there is no way to remove it, I though to remove it on any click in side panel, but then decided to so this. would you prefer do not remove it?